### DISCUSSION TITLE
posts/build-automatic-code-quality-workflow/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#33] [Announcements]
# posts/build-automatic-code-quality-workflow/

代码质量是程序员的职业底线，维护底线不能全靠自觉，本文将介绍如何将 flake8、black、pre-commit 等工具集成到日常的工作流中，省心省力的持续保障代码质量。

https://waynerv.com/posts/build-automatic-code-quality-workflow/

**Comment by monsterxcn** [comment]
好详细！本来只是在找 `pre-commit run --all-files` 来着，没想到被文章细致的小标题和配图吸引了，很 nice 🤣
